src/bilder/images/edxapp/deploy.py

```diff
@@ -101,6 +112,14 @@
     ),
 ]
 if node_type == WEB_NODE_TYPE:
+    vector.configuration_templates.update(
+        {
+            Path(__file__).parent.joinpath("templates", "vector", "nginx.yaml"): {},
+            Path(__file__).parent.joinpath(
+                "templates", "vector", "edx_tracking.yaml"
```

Comment:
How about cms and lms logs?

Reply:
Those are part of the default set of configs that are added in unconditionally.